main.py

```diff
@@ -1,10 +1,49 @@
 import init_django_orm  # noqa: F401
 
+from json import load
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_info:
+        players_data = load(players_info)
+
+    for player, player_info in players_data.items():
+        race_info = player_info.get("race")
+        guild_info = player_info.get("guild")
+        skills_info = race_info.get("skills") if race_info else None
+
+        if race_info:
+            race, _ = Race.objects.get_or_create(
+                name=race_info["name"],
+                description=race_info["description"]
+            )
+        else:
+            race = None
```

Comment:
```suggestion
        race = None
        if race_info:
            race, _ = Race.objects.get_or_create(
                name=race_info["name"],
                description=race_info["description"]
            )
```
More better to read instead of if-else